python/ray/examples/cython_examples

```diff
@@ -0,0 +1 @@
+../../../doc/examples/cython/cython_examples
```

Comment:
Is there a way to avoid symlinks here? They cause problems for people once in a while because people edit them like text files accidentally. This happens most easily on Windows (sine Git disables symlinks by default) but I've seen it happen on non-Windows systems too (I guess people edit directly on GitHub). If we can avoid symlinks without too much trouble that'd be preferable.

Reply:
I'm not sure, as it's a documentation piece that specifically has a test in `python/ray/tests` -- either (a) we rewrite the docs to point to this example living in the main code, (b) we move the test to join the code or (c) some symlink like this. I actually prefer (b) if that's a reasonable answer to you.

Reply:
I don't actually know enough to tell—you'd have to ask someone who's been involved on the code. Not a big deal either way, only worth worrying about if it's not much trouble.

Reply:
yeah, it's not much trouble. and since there's a `setup.py` for the cython_examples already, I'm calling that a separate package (now with its own test)